File: DownstreamService/src/main/java/com/stackroute/DownstreamService/domain/EducationSection.java
package com.stackroute.DownstreamService.domain;


import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class EducationSection {
    private String sectionId;
    private String userId;
    private String operationType;
    private Chicklets[] chicklets;

}

File: angular/src/app/add-module/education-dialog/domain/educationsection.ts
import { EducationChicklets } from './educationchicklets';

export class EducationSection {
    constructor(private sectionId: string,
        private userId: string,
        private operationType: string,
        private chicklets: EducationChicklets[]) {
        this.sectionId = sectionId;
        this.userId = userId;
        this.operationType = operationType;
        this.chicklets = chicklets;
    }
}